db/migrate/20170421145748_remove_uploads.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveUploads < ActiveRecord::Migration[5.0]
+  def change
+    drop_table :uploads
+  end
+end
```

Comment:
Note that this is an irreversible migration; you may wish to refactor it to specify that, or otherwise break out the `up` and `down` methods for this one.